shorten

```diff
@@ -27,6 +27,7 @@ case "${book}" in
   precalculus) dest="${DIR}/data/precalculus/short" && script="${DIR}/books/precalculus/shorten";;
   sociology) dest="${DIR}/data/sociology/short" && script="${DIR}/books/sociology/shorten";;
   u-physics) dest="${DIR}/data/u-physics/short" && script="${DIR}/books/u-physics/shorten";;
+  philosophy) dest="${DIR}/data/philosophy/short" && script="${DIR}/books/philosophy/shorten";;
```

Comment:
i think this line & the empty shorten file can be removed if there isn't a shorten script!

Reply:
oop thank you lol